src/main/java/com/thebois/ColonyManagement.java

```diff
@@ -14,14 +16,19 @@
     private ShapeRenderer batch;
     private BitmapFont font;
     private GameScreen gameScreen;
+    private World world;
+    private WorldView worldView;
+    private final int worldSize = 50;
 
     @Override
     public void create() {
         batch = new ShapeRenderer();
 
         // use libGDX's default Arial font
         font = new BitmapFont();
-        gameScreen = new GameScreen(batch);
+        world = new World(worldSize);
+        worldView = new WorldView(world.getWorld());
+        gameScreen = new GameScreen(batch, worldView);
```

Comment:
Since this method later on will have a lot of different objects constructed in it, we can try to structure it somehow like:

```java
// Models

// Views

// Screens

// Controllers
```

Reply:
Yeah that is probaly nice to have for future us!